### ISSUE TITLE
Compatibility for Colorblind circuit network

### ISSUE THREAD

**Issue by Zehir** [daydev/FactorioWireShortcuts#6]
Hello,

This is a awesome mod, it's really handy to use. But as a colorblind person I got issue with this mod because on the screen it's blue but the icon is red.
Is it possible to add a patch to handle the case where the mod is enabled ?

https://mods.factorio.com/mod/ColorblindCircuitNetwork

Thanks

**Comment by Zehir** [comment]
Or better this updated one : 
https://mods.factorio.com/mod/CustomCircuitColors

**Comment by daydev** [comment]
I don't want to seem presumptuous, but wouldn't it make more sense for that mod to take this one into account? From the description, it seems it has lot of options of colors for all wires, I'll have to basically reimplement their system anew vs. them extending their system to also edit the Wire Shortcuts icons.

**Comment by Zehir** [comment]
Yes make sens this can be added in both mod but yours has been updated in the last 4 years when I found the first one x)
Maybe you can just reuse the item icon is it's was changed ?

**Comment by daydev** [comment]
The first mod you linked has just hard coded changed colors, but the second one has a whole system where you can pick from many colors. I could do just alternative icons with different colors like in the first mod, but since there are different types of color blindness, implementing a free color choice like in the second mod would be pretty involved. How about I add letters or symbols in the corner of the Wire Shortcuts icons for the color blind mode so that you can still tell them apart? Would that be agreeable?

**Comment by Zehir** [comment]
The issue is that the button don't match the icon. Usualy I pick 2 colors that I can easly differenciate like orange and blue. I will ask the other mod author to see what we can do

**Comment by oorzkws** [comment]
@daydev Do you have the original wire-cutter graphics in an editable format? I'm done the implementation on my mod's side, but right now I have to override the graphics for the cutter tool to be something that vaguely communicates what color of wires are being cut.

**Comment by daydev** [comment]
@oorzkws I don't know if you can work with this, but I made my cutter icons from [this file](https://drive.google.com/file/d/1mFl1ATEdNpc0w-_yYROvYhKMKEkSbXzj/view?usp=sharing), toggling layers on and off and exporting.

**Comment by oorzkws** [comment]
> @oorzkws I don't know if you can work with this, but I made my cutter icons from [this file](https://drive.google.com/file/d/1mFl1ATEdNpc0w-_yYROvYhKMKEkSbXzj/view?usp=sharing), toggling layers on and off and exporting.

Thanks. I don't have a GIMP install handy, so I'll have to give that a peek another time. It's on my to-do list now.

**Comment by daydev** [comment]
Which format would work for you? I could try to re-export in GIMP, if available.

**Comment by oorzkws** [comment]
It it can do PSD, that should be fine. TIFF does layers too, I think.

**Comment by daydev** [comment]
@oorzkws  Ok, here's what GIMP could do: [psd](https://drive.google.com/file/d/1A2WJya5f4Qzu3AEgXyk781t56vLo3ers/view?usp=share_link), [tiff](https://drive.google.com/file/d/19wVZcg_myTSDAKabWDcYtbEmLjwuyuxp/view?usp=share_link). I don't have Photoshop to check, but the tiff one appears alright if opened back with GIMP.
